test/test_result.js

```diff
@@ -2,11 +2,13 @@ var expect = require('chai').expect;
 var fs = require('fs');
 
 var Parser = require('../lib/parser');
+var Result = require('../lib/result');
 
 describe('result', function(){
     it('creation', function(done){
         var data = fs.readFileSync('./test/fixtures/valid_result.xml', {encoding: 'utf8'});
         (new Parser()).parse(data, function(error, result){
+            expect(result).to.be.instanceof(Result);
```

Comment:
Don't think this is needed here, the point is not to check the return value of the parser but to assert that we find certain properties on the Result type (this is a test for `Result`).